core/openapi/schemas/language.yaml

```diff
@@ -0,0 +1,34 @@
+type: object
+description:
+  The language used for textual values in this record.
+required:
+  - code
+properties:
+  code:
+    type: string
+    description:
+      The language tag as per RFC-5646.
+    example:
+      code: el
+    name:
+      type: string
+      description:
+        The untranslated name of of the language.
+      example:
+        name: Ελληνικά
+    alternate:
```

Comment:
We recently got a long comment from the STAC community that "alternate" is pretty American and "alternative" would be the better alternative as it's better to understand. Thoughts? Should we switch to alternative?

See https://github.com/stac-extensions/alternate-assets/issues/9

Reply:
I just realized that the rel-type is also alternate, so let's keep it consistent.